src/services/subgraph/index.ts

```diff
@@ -28,7 +28,7 @@ export class SubgraphService extends Service {
     this.yearnSubgraphEndpoint = `https://api.thegraph.com/subgraphs/name/${subgraphName}`;
   }
 
-  async fetchQuery(query: string, variables: { [key: string]: any } = {}): Promise<unknown> {
+  async fetchQuery<T>(query: string, variables: { [key: string]: any } = {}): Promise<T> {
```

Comment:
Made this generic so we don't have to cast it to a type when using it.

Reply:
love it!

Reply:
Much better! Thanks for adding this